resources/qml/pages/MessagingPage.qml

```diff
@@ -336,29 +447,47 @@ Page {
                     }
 
                     if (msgToSend.length > 0) {
-                        shmoose.sendMessage(conversationId, msgToSend, "text");
+                        shmoose.sendMessage(conversationId, msgToSend, "txt", "");
```

Comment:
There are only 3 parameters to that method.

Reply:
Fixed